tools/cmdline.h

```diff
@@ -41,7 +41,7 @@ class CommandLineParser {
     virtual bool matched() const = 0;
 
     // Returns whether this option matches the passed command line argument.
```

Comment:
Please explain what's the meaning of `parse_options` in the comment.

Reply:
Done.